src/lib/tls/cache.c

```diff
@@ -1265,9 +1265,10 @@ static SSL_TICKET_RETURN tls_cache_session_ticket_app_data_get(SSL *ssl, SSL_SES
 	switch (status) {
 	case SSL_TICKET_EMPTY:
 	case SSL_TICKET_NO_DECRYPT:
-#ifdef __clang_analyzer__
+	case SSL_TICKET_FATAL_ERR_MALLOC:
+	case SSL_TICKET_FATAL_ERR_OTHER:
+	case SSL_TICKET_NONE:
 	default:
```

Comment:
The idef should be retained around default, or default should be omitted.  That tells us when new error codes are added that we might need to deal with.

Reply:
Makes sense; so be it. That is liable to mean having to think of another way to placate coverity, so I will make the change and consider what to do about that.